hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/NomisService.kt

```diff
@@ -43,6 +44,14 @@ class NomisService(
   fun getOffender(offenderNo: String): OffenderBooking? = prisonApiWebClient.get()
     .uri("/api/offenders/{offenderNo}", offenderNo)
     .retrieve()
+    .bodyToMono(OffenderBookingOld::class.java)
+    .onErrorResume(NotFound::class.java) { Mono.empty() }
+    .block()
+    ?.toOffenderBooking()
+
+  fun getOffenderNewEndpoint(offenderNo: String): OffenderBooking? = prisonApiWebClient.get()
+    .uri("/api/prisoner-search/offenders/{offenderNo}", offenderNo)
+    .retrieve()
     .bodyToMono(OffenderBooking::class.java)
     .onErrorResume(NotFound::class.java) { Mono.empty() }
     .block()
```

Comment:
Until we move to the new endpoint we need these functions to return the same data - this will allow us to run a parallel test of the old and new endpoints.